zerver/lib/webhooks/git.py

```diff
@@ -35,17 +37,33 @@
 TAG_WITH_URL_TEMPLATE = u"[{tag_name}]({tag_url})"
 TAG_WITHOUT_URL_TEMPLATE = u"{tag_name}"
 
+
 def get_push_commits_event_message(user_name, compare_url, branch_name, commits_data, is_truncated=False):
     # type: (Text, Optional[Text], Text, List[Dict[str, Any]], Optional[bool]) -> Text
+
+    committers = get_all_committers(commits_data)
+    committers_items = list(sorted(committers.items()))  # type: List[Tuple[str, int]]
```

Comment:
the order in which the names of different committers show up in python2 and python3 are different so to pass both their backend tests i used `sorted`.
in python3 i get an error which does not allow indexing of dictionary items so i am convrting it to `list`